db/schema.rb

```diff
@@ -10,14 +10,24 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-
-ActiveRecord::Schema.define(version: 20170405205648) do
-
+ActiveRecord::Schema.define(version: 20170406184924) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "clients", force: :cascade do |t|
+    t.integer  "user_id"
+    t.string   "first_name"
+    t.string   "last_name"
+    t.string   "street"
+    t.string   "city"
+    t.string   "state"
+    t.string   "zip"
```

Comment:
Given that it may include dashes and the regex prevents non-numeric entries, I think having it as a string makes more sense - we're unlikely to need to manipulate the zip as an integer (ie. do math etc)